src/theme/walletConnectModalCSS.tsx

```diff
@@ -0,0 +1,10 @@
+import { css } from 'styled-components'
+
+export const reactWalletConnectModalCSS = css`
```

Comment:
The only thing would be to remove the react part as the lib is just wallet connect. The rest is fine!

Reply:
Done! Thanks